### PR TITLE
Updated Service Account Manager documentation

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR:
- updates notes in various topics
- renames Project Namespace Access to Service Account Manager for clarity
- fixes broken links

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: docs/concepts/controller-manager.md [hunk 1]
```diff
@@ -98,8 +98,10 @@ In order to activate it, provide the following configuration:
 * `concurrentSyncs`: The amount of goroutines scheduled for reconciling events.
 * `ttlNonShootEvents`: When an event reaches this time-to-live it gets deleted unless it is a Shoot-related event (defaults to `1h`, equivalent to the `event-ttl` default).
 
-> :warning: In addition, you should also configure the `--event-ttl` for the kube-apiserver to define an upper-limit of how long Shoot-related events should be stored.
+{{% alert color="warning"  title="Warning" %}}
```
Comment (rfranzke) [inline docs/concepts/controller-manager.md]: What is this special syntax? Can we please retain the markdown syntax?

Comment (n-boshnakov) [reply docs/concepts/controller-manager.md]: We have moved away from using the markdown syntax in the documentation due to the increased visibility and functionality the shortcodes offer in the website. 

Here is how the rendered markdown looks:
Screenshot 2022-11-03 111057
And here is how the rendered shortcode:
Screenshot 2022-11-03 111229

You can find more information on the subject of Shortcodes [in the documentation](https://gardener.cloud/docs/contribute/20_documentation/30_shortcodes/).

Comment (rfranzke) [reply docs/concepts/controller-manager.md]: Given that all these docs are written and maintained by developers and we don't have dedicated folks just for the documentation, I doubt that anything else other than Markdown-only is a sustainable way. You don't want to run behind all these changes and adapt it again and again.
Personally, I don't see much difference in the display, so the impact of remaining with Markdown seems rather minimal.

Comment (n-boshnakov) [reply docs/concepts/controller-manager.md]: Alright. I will revert the shortcode change.

File: docs/usage/service-account-manager.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
-# Project Namespace Access
+# Service Account Manager
 
-## Service Account Manager
+## Overview
 With Gardener `v1.47` a new role called `serviceaccountmanager` was introduced. This role allows to fully manage `ServiceAccount`'s in the project namespace and request tokens for them. This is the preferred way of managing the access to a project namespace as it aims to replace the usage of the default `ServiceAccount` secrets that will no longer be generated automatically with Kubernetes `v1.24+`.
 
+## Actions
+
+Once given the `serviceaccountmanager` role, a user can create/update/delete `ServiceAccount`s in the project namespace.
```
Comment (rfranzke) [inline docs/usage/service-account-manager.md]:
```suggestion
Once assigned the `serviceaccountmanager` role, a user can create/update/delete `ServiceAccount`s in the project namespace.
```

**Review by rfranzke — COMMENTED** [review]
Thanks! Small nit, otherwise lgtm

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve
/milestone v1.59

**Pull request merged**